pylibczi/CziFile.py

```diff
@@ -46,20 +50,75 @@ class CziFile(object):
     #   units for the scale in the xml file are not correct (says microns, given in meters)
     scale_units = 1e6
 
-    def __init__(self, czi_filename, metafile_out='', use_pylibczi=True, verbose=False):
-        self.czi_filename = czi_filename
+    # Dims as defined in libCZI
```

Comment:
May be nice to add a link to the CZI docs that explain what each dimension mean in more depth while you are at it?